.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Just remove the .idea folder, guess was already added before you added it to the .gitignore